### PR TITLE
Fix part of #138: Topic train fragment Low-fi UI (Part 1)

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Replicated from #197 

Part 1: This PR contains basic XML files for creating TopicTrainFragment. 
Part 2: #202
Part 3: #203
Part 4: #204 

Mock: https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/f403add1-079a-4202-9975-efe04c076290/Home-Page-1-Start-Learning-Pre-Scroll-6

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is **assigned** to an appropriate reviewer.

### CODE REVIEW

**Comment by rt4914** [comment]
@veena14cs @nikitamarysolomanpvt PTAL

File: app/build.gradle [hunk 1]
```diff
@@ -61,6 +61,7 @@ dependencies {
       'androidx.constraintlayout:constraintlayout:1.1.3',
       'androidx.core:core-ktx:1.0.2',
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
+      'androidx.recyclerview:recyclerview:1.0.0',
       'com.google.dagger:dagger:2.24',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
       "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1",
```
Comment (veena14cs) [inline app/build.gradle:67]: use single quotes

**Review by veena14cs — APPROVED** [review]
LGTM

File: app/build.gradle (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/build.gradle:67]: single quotes not yet updated

**Review by nikitamarysolomanpvt — COMMENTED** [review]
previous suggestions are not updated in split PR

File: app/build.gradle (hunk 1 above)
Comment (rt4914) [reply app/build.gradle:67]: Actually, I mentioned this in my last PR (#197 ), I am only adding a new recyclerview and I prefer not to update someone else's PR. The double quotes has been mentioned from Ben's PR.

Later on we can solve such nit issues by creating a separate PR.

**Comment by rt4914** [comment]
Merging this PR, as it looks up-to-date.

File: app/build.gradle (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/build.gradle:67]: Okay

**Pull request merged**